app/src/main/java/com/scouts/kitchenplaner/ui/navigation/NavGraphProjectCreation.kt

```diff
@@ -47,14 +48,8 @@ fun NavGraphBuilder.projectCreationNav(navController: NavHostController) {
             InviteToProject (
                 projectId = projectId,
                 onNavigateToProject = {
-                    navController.navigate(
```

Comment:
For me it is strange that when navigating back I'm not again at the project creation screen. And I do not get any information whether the project was created.

Reply:
The issue was that the project was actually already created and thus if you navigated back to the project creation screen you were automatically redirected to the Invitation Screen again (i.e. you were trapped in that screen). This solves that issue. Perhaps the text on the Invitation screen should be changed to something like "Das Projekt wurde erstellt. Möchtest du andere Leute dazu einladen?" to clarify that the project creation was succesful?

Reply:
That sounds good.